x-pack/legacy/plugins/siem/public/pages/home/home_navigations.tsx

```diff
@@ -54,7 +54,7 @@ export const navTabs: SiemNavTab = {
   [SiemPageName.case]: {
     id: SiemPageName.case,
     name: i18n.CASE,
-    href: getCaseUrl(),
+    href: getCaseUrl(null),
```

Comment:
Considering changing it into an optional param

Reply:
I appreciate why it feels like making the parameter optional would be more ergonomic, and thanks for the suggestion! At first, I felt the same way while reviewing similar code:

While reviewing a recent PR, I found some inconsistencies in URL navigation behavior. To the best of my current knowledge, we want to preserve URL state in links in all but one case: the global nav links.

Since in all other cases we want the caller to provide a value, I suggested we force the caller to provide it as a parameter, or explicitly opt out with by passing a `null`.

If your ok with it, I'd rather not make it optional until we have the team discussion / agreement @XavierM suggested here: https://github.com/elastic/kibana/pull/60348#discussion_r393431979